src/main/java/net/javadiscord/javabot/data/config/guild/ModerationConfig.java

```diff
@@ -96,4 +98,8 @@ public Role getAdminRole() {
 	public Role getExpertRole() {
 		return this.getGuild().getRoleById(this.expertRoleId);
 	}
+
+	public long getMinActiveHoursToSendMedia() {
```

Comment:
Remove this, we have made custom getters for Channels & Roles to return their respective objects instead of plain IDs. This is already done by the `@Data` annotation